generators/aws-containers/prompts.js

```diff
@@ -200,32 +222,31 @@ function askPerformances() {
 function promptPerformance(config, awsConfig = { performance: 'low' }) {
     if (this.abort) return null;
 
-    const performanceLevels = _.keys(PERF_TO_CONFIG)
+    const prodDatabaseType = config.prodDatabaseType;
+
+    if(prodDatabaseType === databaseTypes.postgresql) {
+        this.log.ok(` ⚠️ Postgresql databases are currently only supported by Aurora on high-performance database instances`);
+    }
+
+    const performanceLevels = _(PERF_TO_CONFIG).keys()
         .map((key) => {
             const perf = PERF_TO_CONFIG[key];
-            return {
-                name: `${_.startCase(key)} Performance \t ${chalk.green(`Task: ${perf.fargate.CPU} CPU Units, ${perf.fargate.memory} Ram, Count: ${perf.fargate.taskCount}`)}\t ${chalk.yellow(`DB: ${perf.database.instances} Instance, Size: ${perf.database.size}`)}`,
-                value: key,
-                short: key
-            };
-        });
-
+            if (perf.database.supportedEngines.includes(prodDatabaseType)) {
```

Comment:
what is the `else` return value?

Reply:
Also, could you put the check `perf.database.supportedEngines.includes(prodDatabaseType)` in a variable?
I'm thinking `isEngineSupported`

Reply:
🌮